app/forms/instance-create.tsx

```diff
@@ -244,10 +248,10 @@ export function CreateInstanceForm() {
   const imageSizeGiB = image?.size ? Math.ceil(image.size / GiB) : undefined
 
   useEffect(() => {
-    if (createInstance.error) {
+    if (error) {
       setIsSubmitting(false)
     }
-  }, [createInstance.error])
```

Comment:
This one was not actually a #2367 problem because the dep array is looking at the same `error` value in both the original and the updated version. So in this case I think the vagueness of the name `error` might weigh against this change. I'd say if you want to keep it anyway, rename `error` to `mutationError`. `isPending` is ok to me because RQ is the only thing that uses that word.

Reply:
Good point; no strong feelings on this one, so will close it.